src/amo/components/Link.js

```diff
@@ -18,6 +21,9 @@ export class LinkBase extends React.Component {
   }
 
   static defaultProps = {
+    // TODO: Make this default to `true` and force internal links to label
+    // themselves as not external?
```

Comment:
Well it would only be for links using `href`, as all internal links use `to` which links them in React Router. But then it's likely a hassle regardless... we can cross this bridge later if we want I'd say.